src/get_taxa.Rmd

```diff
@@ -189,10 +208,102 @@ Total number of taxa with valid distributions:
 nrow(checklist_taxa)
 ```
 
+# Get backbone info
+
+To make decisions regarding synonyms and higher classification, we need to retrieve GBIF backbone information, if present:
+
+
+```{r retrieve_taxonomic_info_backbone}
+taxon <- checklist_taxa %>% filter(is.finite(nubKey))
+nub_keys <- checklist_taxa %>% 
+  filter(is.finite(nubKey)) %>% 
+  distinct(nubKey)
+progress_bar <- progress_estimated(nrow(nub_keys))
+taxon_backbone_info <-  map_dfr(nub_keys$nubKey, ~ {
+  progress_bar$tick()$print()
+  rgbif::name_usage(key = ., return = "data")
+  })
+```
+
+We select taxonomic information we are interested to:
+
+```{r select_cols_backbone}
+taxon_info <- c("key", "scientificName", "species", "genus", "family", "order",
+                "class", "phylum", "kingdom",
+                "rank", "speciesKey", "taxonomicStatus", 
+                "acceptedKey", "accepted", "issues")
+
+taxon_backbone_info <- taxon_backbone_info %>%
+  select(one_of(taxon_info))
+```
+
+In case some information misses, add empty column(s):
```

Comment:
@damianooldoni why is this step necessary?

Reply:
Do you refer to the step of adding empty columns?

Reply:
Indeed, I've deleted the step.